src/Forms/DatetimeField.php

```diff
@@ -181,6 +180,13 @@ public function localisedToISO8601($datetime)
         }
         $fromFormatter = $this->getFormatter();
         $toFormatter = $this->getISO8601Formatter();
+
+        // Remove 'T' date and time separator before parsing (required by W3C HTML5 fields)
+        if ($this->getHTML5()) {
```

Comment:
Ignore this, @chillu has indicated this would allow invalid dates to be passed in.